tests/init.go

```diff
@@ -227,7 +227,7 @@ var Forks = map[string]*params.ChainConfig{
 		MergeNetsplitBlock:      big.NewInt(0),
 		TerminalTotalDifficulty: big.NewInt(0),
 	},
-	"ArrowGlacierToMergeAtDiffC0000": {
+	"Shanghai": {
```

Comment:
Please keep `ArrowGlacierToMergeAtDiffC0000` as is and introduce `Shanghai` as a new element (copy `Merge` and add `ShanghaiTime`)

Reply:
Resolved by [581f18f](https://github.com/ledgerwatch/erigon/pull/6215/commits/581f18ffb84918667677e8ae1717aaa874ab1325)